README.md

```diff
@@ -173,6 +175,8 @@ If you have questions or comments, please create an issue.
 - [A Tutorial Introduction to Git](https://git-scm.com/docs/gittutorial) - A Tutorial for Beginners by Git.
 - [First Aid Git](https://firstaidgit.io/#/) - A searchable collection of the most frequently asked Git questions. Answers for these questions were collected from personal experience, Stackoverflow, and the official Git documentation.
 - [Git by Susan Potter](https://www.aosabook.org/en/git.html) - Show how various technical aspects of Git work under the covers to enable distributed workflows, and how it differs from other version control systems (VCSs).
+- ["Git && Github Complete - by Love Babbar Youtube Channel"](https://www.youtube.com/watch?v=uj4fy4kpaOA) - A hindi video tutorial for beginners to understand how Git and Github works.
```

Comment:
Thanks for sharing this! Because this is in Hindi, I think removing these and adding them to the Hindi README file (https://github.com/freeCodeCamp/how-to-contribute-to-open-source/blob/main/README-HI.md) would be more appropriate.

Reply:
I'm sorry but most of the Hindi speaking people prefer to read in English. So they're going to see the English readme file and open the link to the Hindi tutorial. So I think it's appropriate.